types/src/proof/definition.rs

```diff
@@ -28,6 +28,11 @@ pub struct AccumulatorProof {
     siblings: Vec<HashValue>,
 }
 
+/// Because leaves can only take half the space in the tree, any numbering of the tree leaves
+/// must not take the full width of the total space.  Thus, for a 64-bit ordering, our maximumm
+/// proof depth is limited to 63.
+pub const MAX_PROOF_DEPTH: usize = 63;
```

Comment:
Should we name this something similar to `MAX_ACCUMULATOR_PROOF_DEPTH` because it doesn't apply to the state tree?

Reply:
Good point, done.